src/admin-views/settings/featured/description.php

```diff
@@ -0,0 +1,18 @@
+<?
+/**
+ * Title for featured settings box.
+ *
+ * @since TBD
+ *
+ * @var Tribe__Tickets__Admin__Views $this           Template object.
+ * @var string                       $description    Featured settings description.
+ */
+
+if( empty( $description ) ) {
+    return;
+}
+
+?>
+<div class="tec-tickets__admin-settings-featured-description">
+    <?php echo esc_html( $description ); ?>
```

Comment:
The docblock on the main template mentions that this could have HTML. If we have a link or something here we may want to print it directly and add an ignore to the phpcs rule.

Reply:
Good catch. You're right. It should be flexible enough to display HTML.